api/ocm/tools/transfer/transferhandler/options.go

```diff
@@ -0,0 +1,38 @@
+package transferhandler
+
+import (
+	"github.com/mandelsoft/goutils/errors"
+)
+
+//////////////////////////////////////////////////////////////////////////////
+
+// ConfigOption describes generic non-standard options.
+// Specialized option set implementation maps such generic
+// config to their specialized settings. The format depends
+// on the option target. For example, for spiff it is a spiff
+// script.
+type ConfigOption interface {
+	SetConfig([]byte)
+	GetConfig() []byte
+}
+
+type configOption struct {
+	config []byte
+}
```

Comment:
I feel like it is confusing that the type `configOption` is not the type that's supposed to be implementing the interface `ConfigOption`.

Reply:
It is supposed as option (like the other struct). In this case an option to feed the ConfigOption interface.

Reply:
Renamed to `ConfigOptionConsumer` and added more docu.